packages/web-components/src/select/select.styles.ts

```diff
@@ -208,57 +226,40 @@ export const selectStyles = (context, definition) =>
     appearanceBehavior('stealth', selectStealthStyles(context, definition)),
     forcedColorsStylesheetBehavior(
       css`
-        :host([disabled]) {
-          border-color: ${SystemColors.GrayText};
-          background-color: ${SystemColors.ButtonFace};
-          color: ${SystemColors.GrayText};
-          opacity: 1;
-          forced-color-adjust: none;
-        }
-
-        :host([disabled]:hover) {
-          background: ${SystemColors.ButtonFace};
-        }
-
-        :host([disabled]) .control {
-          color: ${SystemColors.GrayText};
-          border-color: ${SystemColors.GrayText};
-        }
-
-        :host(:not([disabled]):hover) {
-          background: ${SystemColors.ButtonFace};
-          border-color: ${SystemColors.Highlight};
-        }
-
-        :host(:${focusVisible}) {
-          forced-color-adjust: none;
-          background: ${SystemColors.ButtonFace};
-          border-color: ${SystemColors.Highlight};
-          box-shadow: 0 0 0 1px inset ${SystemColors.Highlight};
-          color: ${SystemColors.ButtonText};
-          fill: currentcolor;
-        }
-
-        :host([open]) .listbox {
-          background: ${SystemColors.ButtonFace};
-          border: 1px solid ${SystemColors.ButtonText};
-        }
-
-        :host(:${focusVisible}) ::slotted([aria-selected="true"][role="option"]:not([disabled])) {
-          background: ${SystemColors.Highlight};
-          border-color: ${SystemColors.ButtonText};
-          box-shadow: 0 0 0 calc(${focusStrokeWidth} * 1px) inset ${SystemColors.HighlightText};
-          color: ${SystemColors.HighlightText};
-          fill: currentcolor;
-        }
-
-        ::slotted([role='option']:not([aria-selected='true']):not([disabled]):hover) {
-          forced-color-adjust: none;
-          color: ${SystemColors.ButtonText};
-          background: ${SystemColors.ButtonFace};
-          border-color: ${SystemColors.Highlight};
-          box-shadow: none;
-        }
-      `,
-    ),
-  );
+      :host {
+        background: ${SystemColors.ButtonFace};
+        color: ${SystemColors.ButtonText};
+        fill: currentcolor;
```

Comment:
Fill is already `currentcolor` in main styles. Does this need to be set again in HC? Seems like in both regular and HC we always have the fill set to match color, and it would be good if we never had to set that specifically in the forced colors model.

Reply:
correct, currentcolor not needed here